.studio/secrets-service

```diff
@@ -215,7 +215,7 @@ function secrets_test_A1_migration() {
   hab svc start $PKG_IDENT
 
   # Wait for the secrets-service to come online
-  wait_for_svc_to_load $PKG_IDENT
+  wait_or_fail_for_svc_to_load $PKG_IDENT
```

Comment:
`wait_for_svc_to_load` was actually deprecated in studio-common, so I guess very few people are calling this function.